packages/widgetbook/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - **FIX**: Skip decoding non-ascii characters in URLs. ([#1218](https://github.com/widgetbook/widgetbook/pull/1218) - by [@shigomany](https://github.com/shigomany))
 - **FIX**: Encode all fields values to allow reserved characters (e.g. commas, colons and curly brackets). ([#1214](https://github.com/widgetbook/widgetbook/pull/1214))
+- **Feat**: Allow custom theming/personalization of widgetbook. ([#1225](https://github.com/widgetbook/widgetbook/pull/1225))
```

Comment:
**suggestion:** Because you have a lot of contributions, you need to know how we define our CHANGELOG entries:

1. `FEAT` entries should be the first in the list, before the `FIX` entries.
1. For external PRs, you should mention yourself next to the PR number.
1. "Widgetbook" should start with a capital letter, when referred to.

```suggestion
- **FEAT**: Allow changing Widgetbook's theme and mode. ([#1225](https://github.com/widgetbook/widgetbook/pull/1225) - by [@Mastersam07](https://github.com/Mastersam07))
```